google/cloud/spanner/value.cc

```diff
@@ -88,6 +88,83 @@ bool Equal(google::spanner::v1::Type const& pt1,
   }
 }
 
+std::ostream& StreamHelper(std::ostream& os, google::protobuf::Value const& v,
+                           google::spanner::v1::Type const& t) {
+  if (v.kind_case() == google::protobuf::Value::kNullValue &&
+      t.code() != google::spanner::v1::ARRAY &&
+      t.code() != google::spanner::v1::STRUCT) {
+    return os << "";
```

Comment:
Why stream the `""`?  Did you mean `"NULL"`?

Reply:
Done.